replication.go

```diff
@@ -24,10 +24,31 @@ package driver
 
 import (
 	"context"
+	"time"
 )
 
+// Tick is represent a place in either the Write-Ahead Log,
+// journals and datafiles value reported by the server
+type Tick string
+
+// Batch represents state on the server used during
+// certain replication operations to keep state required
+// by the client (such as Write-Ahead Log, inventory and data-files)
+type Batch interface {
+	// id of this batch
+	BatchID() string
+	// Last tick reported by this batch
+	Tick() Tick
```

Comment:
If it returns the last tick, then I would prefer this function to be called `LastTick`